.babelrc

```diff
@@ -1,3 +1,17 @@
 {
-  "presets": ["react-native"]
-}
+  "retainLines": true,
+  "compact": true,
+  "comments": false,
+  "presets": [
+    "react-native",
+    "react-native-stage-0/decorator-support"
+  ],
+  "plugins": [
+    ["relay", {
+      "schema": "data/schema.json",
+      "debug": true,
+      "suppressWarnings": false
+    }]
+  ],
+  "sourceMaps": false
+}
```

Comment:
did this worked?

Reply:
I think so! It didn't break like the other one 😅

Reply:
nice